src/scss/_grid.scss

```diff
@@ -0,0 +1,10 @@
+.grid {
```

Comment:
How might you build out your grid a bit more? Have you looked into the `::last-of-type` selector? Check out more about it here: https://css-tricks.com/dont-overthink-it-grids/